docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:20.17.0
+WORKDIR /usr/app
```

Comment:
/usr/share/app might be better?
Also is there a reason we need to copy package*, install node_modules, then copy everything else over here?